experimental/tau_value_determination.py

```diff
@@ -163,7 +208,7 @@ def draw_l_shape( l_shape, best_tau, x_value, y_value, channel, variable ):
         plt.savefig( 'plots/tau_from_L_shape_%s_channel_%s_MC.png' % ( channel, variable ) )
 
 def get_data_histogram( channel, variable, met_type ):
-    fit_result_input = '../../data/8TeV/%(variable)s/fit_results/central/fit_results_%(channel)s_%(met_type)s.txt'
+    fit_result_input = 'data/absolute_eta_M3_angle_bl/8TeV/%(variable)s/fit_results/central/fit_results_%(channel)s_%(met_type)s.txt'
```

Comment:
That should be covered by ```src/unfolding_tests/get_best_regularisation.py``` which supersedes ```experimental/tau_value_determination.py ```. Have a look at ``` config/unfolding/MET_7TeV_electron_channel.json```. It uses the fit normalisation for data input. 
I should probably fix the other JSON files as they seem to use the unfolding ROOT files.